components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/ImportUtils.java

```diff
@@ -1059,6 +1185,77 @@ public static String getFileContentAsJson(String pathToArchive) throws IOExcepti
         return jsonContent;
     }
 
+    /**
+     * Retrieve Policy Definition as JSON content.
+     *
+     * @param pathToArchive Path to Policy archive
+     * @throws IOException If an error occurs while reading the file
+     */
+    public static String getPolicyFileContentAsJson(String pathToArchive) throws IOException {
+
+        String jsonContent = null;
+        String pathToYamlFile = pathToArchive + ImportExportConstants.YAML_EXTENSION;
+        String pathToJsonFile = pathToArchive + ImportExportConstants.JSON_EXTENSION;
+
+        // Load yaml representation first if it is present
+        if (CommonUtil.checkFileExistence(pathToYamlFile)) {
+            if (log.isDebugEnabled()) {
+                log.debug("Found policy definition file " + pathToYamlFile);
+            }
+            String yamlContent = FileUtils.readFileToString(new File(pathToYamlFile));
+
+            jsonContent = CommonUtil.yamlToJson(yamlContent);
+
+            jsonContent = getJsonContentWithData(jsonContent);
+
```

Comment:
Shall we remove these unnecessary new lines?

Reply:
Fixed by [d413768](https://github.com/wso2/carbon-apimgt/pull/11325/commits/d41376893b76bc5aaafe2e28c65b6da4dd2d5642)